examples/todolist/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM node:6.1
+
+# Create app directory
+RUN mkdir -p /usr/src/app
```

Comment:
`WORKDIR` creates directory (https://docs.docker.com/engine/reference/builder/#workdir) - looks like this layer is not required.